here-naksha-app-service/src/test/java/com/here/naksha/app/service/SearchFeaturesTest.java

```diff
@@ -104,28 +105,31 @@ void tc1002_testSearchWrongSpace() throws URISyntaxException, IOException, Inter
   }
 
   @Test
-  void tc1003_testSearchWithTagsAndProp() throws URISyntaxException, InterruptedException, JSONException, IOException {
+  void tc1003_testSearchWithTagsAndProp() throws URISyntaxException, InterruptedException, IOException {
     // Test API : GET /hub/spaces/{spaceId}/search
     // Validate features getting returned for given Tag and Property search conditions
+    // with response displaying only selected properties
 
     // Given: search query
     final String tagsQueryParam = "tags=one+four";
     final String propQueryParam = "p.speedLimit='60'";
-    final String expectedBodyPart = loadFileOrFail("ReadFeatures/Search/TC1003_searchWithTagsAndProp/search_response.json");
     final String streamId = UUID.randomUUID().toString();
+    final String selectionParam = "selection=p.speedLimit,%s".formatted(urlEncoded("p.@ns:com:here:xyz.tags"));
+    final String expectedBodyPart = loadFileOrFail("ReadFeatures/Search/TC1003_searchWithTagsAndProp/search_response.json")
```

Comment:
Same comment as previous one. We need to add few more attributes to original features, otherwise this selection test is not valid.